.eslintrc.js

```diff
@@ -0,0 +1,111 @@
+module.exports = {
+    parserOptions: {
+        ecmaVersion: 2022,
+        sourceType: 'module'
+    },
+    env: {
+        node: true,
+        es6: true,
+        jest: true
+    },
+    plugins: [],
+    extends: ['eslint:recommended', 'plugin:vue/vue3-essential'],
+    // 'off' or 0 - turn the rule off
+    // 'warn' or 1 - turn the rule on as a warning (doesn’t affect exit code)
+    // 'error' or 2 - turn the rule on as an error (exit code will be 1)
+    rules: {
```

Comment:
Might it be better for us to adopt a set of rules (like airbnb) that we can use across our different repos, rather than hand-configuring different rules?

Reply:
For now, I just used my custom set of rules to get started. IMO there are some gaps in most of the default packages such as Airbnb.
We should walk through linting rules in our next dev meeting 👍

Reply:
After we discuss, I can update them in a future commit